modules/ServerAPI/src/client/Experiment.coffee

```diff
@@ -449,7 +449,12 @@ class ExperimentBaseController extends BaseEntityController
 			@setupProjectSelect()
 		@setupAttachFileListController()
 		@setupCustomExperimentMetadataController()
+		# Parse "data column order" states and setup controllers
```

Comment:
Code for rendering the read-only Endpoint Table in the experiment browser and experiment editor.